test_4_filter_numbers.py

```diff
@@ -1,7 +1,8 @@
 def filter_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
-
+  return arr < 10
```

Comment:
I see a couple of issues here. First, on this line you are comparing a List (what Python calls Arrays) against a numerical value. These are two data types that can not be compared.